plio/cache.py

```diff
@@ -3,7 +3,7 @@
 
 
 def get_cache_key(instance):
-    """Calculates the cache key for an instance based on selected schema and """
+    """Calculates the cache key for an instance based on selected tenant schema"""
```

Comment:
since the tenant is take from the connection and not passed as a param
```suggestion
    """Calculates the cache key for an instance based on the current tenant schema"""
```